specs/endpoint-security.spec.yml

```diff
@@ -10,6 +10,8 @@ inputs:
     outputs:
       - elasticsearch
       - logstash
+    agent_actions:
```

Comment:
I like this as a syntax for allow us to specify the action capabilities for inputs, but you need to document how it works. 

The current behaviour is a bit weird though, because this is not an exhaustive list of all the actions endpoint supports. This would make more sense if having this list restricted endpoint to only receiving the actions listed here, and inputs that omitted it received every action.

At the same time I see value in minimizing the scope of what we are changing here. Maybe we just need to rename this to something like `agent_action_features` to indicate that it is more like a feature flag mechanism, and we are only opting in to the additional actions listed here.

Reply:
Over time I can definitely see us wanting to change this into an exhaustive list of supported actions we route to inputs.

I feel like that behaviour is probably the right approach, will get more opinions on this.

Reply:
Yeah, was not sure on the approach. Basically want to have some generic mechanism to allow the component to express it's interest in the particular agent actions (get the agent action forwarded to the component) in this case just needed the signed UNENROLL action to be forwarded to Endpoint.

Reply:
Thanks! Done.